README.md

```diff
@@ -32,15 +32,19 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? 
+I would say it makes it readable for other developers and organizes your html so its not a jumbled mess.
 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+Block encloses content into block sections where inline doesnt it runs it continusly in a line with no line breaks.
+block you also can implement height and width attributes
 
-3. What are the 4 areas of the box model?
+3. What are the 4 areas of the box model?content, padding, margin, border
 
-4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+4. While using flexbox, what axis does the following property work on: ```align-items: center```verticle
```

Comment:
Not necessarily. If `flex-direction` is `row`, then `align-items` would affect the vertical axis. But if it is `column`, it will affect the _horizontal_ axis. This is because `align-<whatever>` is perpendicular to whatever the `flex-direction` is.

A better answer is "cross axis", as the axis determined by flex direction is called the "main axis", and `align-<whatever>` affects the axis perpendicular to that, or in other words, the axis which "crosses" it.